templates/insurance_companies/general_edit.html

```diff
@@ -90,6 +90,14 @@
             </div>
         </div>
     {/if}
+    {if $SUPPORT_ELIGIBILITY_REQUESTS}
+    <div class="form-group">
+        <label for="eligibility_id" class="control-label col-sm-2">{xl t='Payer Id For Eligibility'}</label>
+        <div class="col-sm-8">
+            <input type="text" id="eligibility_id" name="eligibility_id" class="form-control" value="{$insurancecompany->get_eligibility_id()|escape:attr}" onKeyDown="PreventIt(event)">
```

Comment:
Smarty has a attr function now and the above code appears to be a hybrid between new and old code :)
For 5.0.2 code use:
```
<input type="text" id="eligibility_id" name="eligibility_id" class="form-control" value="{$insurancecompany->get_eligibility_id()|attr}" onKeyDown="PreventIt(event)">
```
For the backport, need to use:
```
<input type="text" id="eligibility_id" name="eligibility_id" class="form-control" value="{$insurancecompany->get_eligibility_id()|escape:'html'}" onKeyDown="PreventIt(event)">
```

Reply:
yep, i'm lazy. did the mods in 501 and resolved conflicts in 502!